src/css/components/Drawer.scss

```diff
@@ -0,0 +1,98 @@
+$drawer-box-shadow-offset: 6px;
+$drawer-box-shadow-blur: 12px;
+$drawer-box-shadow-color: rgba(0, 0, 0, 0.2);
+$drawer-box-shadow-blur-color: #{$drawer-box-shadow-blur} #{$drawer-box-shadow-color};
+$drawer-border: 1px solid #ddd;
+
+.Drawer {
+  background-color: #fff;
```

Comment:
We'll probably want to pull this out to variable for themeing purposes. Actually, it would probably be good find common colors for surfaces and pull them out as well. Might be beyond the scope of this, though.

Reply:
Totally agreed. We might want to consider updating our Sass to use `@use` over `@import` at some point in the near future. I'm not sure when they plan on dropping `@import` completely. That might be a good time to tighten up things like this.